src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py

```diff
@@ -734,7 +734,7 @@ def _open(filename, mode):
         private_key = Ed25519PrivateKey.generate()
         private_bytes = private_key.private_bytes(
             encoding=serialization.Encoding.PEM,
-            format=serialization.PrivateFormat.PKCS8,
+            format=serialization.PrivateFormat.OpenSSH,
```

Comment:
It will be helpful to comment with a sample key file, such as what I did in #30063:

```py
        # The private key will look like:
        # -----BEGIN RSA PRIVATE KEY-----
        # ...
        # -----END RSA PRIVATE KEY-----
```

Reply:
`encoding=serialization.Encoding.PEM, format=serialization.PrivateFormat.OpenSSH` generates a private key as:

```
-----BEGIN OPENSSH PRIVATE KEY-----
...
-----END OPENSSH PRIVATE KEY-----
```

`encoding=serialization.Encoding.PEM, format=serialization.PrivateFormat.PKCS8` generates a private key as:

```
-----BEGIN PRIVATE KEY-----
...
-----END PRIVATE KEY-----
```

Reply:
yes, you are correct
the existing feature is generating format OpenSSH